### PR TITLE
Add arm64 to multi-arch migration tests on nightly payloads

### PR DESCRIPTION
**Pull request by shellyyang1989**
Refers [MIXEDARCH-134](https://issues.redhat.com/browse/MIXEDARCH-134) [MIXEDARCH-167](https://issues.redhat.com/browse/MIXEDARCH-167)

### CODE REVIEW

**Comment by shellyyang1989** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-aws-ipi-to-multiarch-p2-f14

**Comment by shellyyang1989** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-aws-ipi-to-multiarch-p2-f14

**Comment by shellyyang1989** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-aws-ipi-to-multiarch-p2-f14

**Comment by shellyyang1989** [comment]
The change includes:
1) arm64 to multi-arch migration on nightly payloads
2) changing the multi-arch release from ec to nightly because latest ec version is lower than latest nightly and downgrade is not what we support.

**Comment by shellyyang1989** [comment]
cc @aleskandro @jianlinliu @liangxia for review and merge. Thanks!

**Review by aleskandro — APPROVED** [review]

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml [hunk 1]
```diff
@@ -866,6 +924,25 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-baselinecaps-v412
+- as: azure-ipi-baselinecaps-v413-to-multiarch-p2-f14
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:927]: The workflow is `cucushift-installer-rehearse-azure-ipi-baselinecaps-v412`, so `v413` looks like a typo here

**Comment by shellyyang1989** [comment]
Seems like there is bug in CVO. Will reproduce later when I get time.

```
# cat openshift-cluster-version/pods/version--xdps4-cgkzw/cleanup/cleanup/logs/current.log
2023-03-02T06:36:32.089843295Z exec /usr/bin/sh: exec format error

initContainerStatuses:
  - containerID: cri-o://8410b9c89aa41b5f07a2804f198965e35bd81120eeb3501520e781b14c4d82c5
    image: registry.build03.ci.openshift.org/ci-op-cqg19xi2/release@sha256:8b141ce490d94d4f7af815e846498266d72c0c14df4c8e60ad1a4a49238753cd
    imageID: registry.build03.ci.openshift.org/ci-op-cqg19xi2/release@sha256:8b141ce490d94d4f7af815e846498266d72c0c14df4c8e60ad1a4a49238753cd
    lastState:
      terminated:
        containerID: cri-o://8410b9c89aa41b5f07a2804f198965e35bd81120eeb3501520e781b14c4d82c5
        exitCode: 1
        finishedAt: "2023-03-02T06:36:32Z"
        reason: Error
        startedAt: "2023-03-02T06:36:32Z"
    name: cleanup
    ready: false
    restartCount: 3
    state:
      waiting:
        message: back-off 40s restarting failed container=cleanup pod=version--xdps4-cgkzw_openshift-cluster-version(fd840362-84c6-4c36-b237-8b5f29dcbcdf)
        reason: CrashLoopBackOff
```

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml (hunk 1 above)
Comment (shellyyang1989) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:927]: ah, right. v413 has not supported yet. Thanks!

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml]: yesterday, we just added v413 support, you can use v413 for testing now, but need to ensure the job title and workflow matched.

Comment (shellyyang1989) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml]: Updated!

**Pull request merged**